data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.py

```diff
@@ -62,19 +79,23 @@ def fetch_artic_primers(output_directory, primers):
             )
             exit(response.status_code)
         bed_output_filename = os.path.join(output_directory, name + ".bed")
-        write_artic_style_bed(StringIO(response.text), bed_output_filename)
-        description = name[:-2] + " " + name[-2:] + " primer set"
+        convert_and_write_bed(StringIO(response.text), bed_output_filename, name)
+        if 'ARTIC' in name:
+            # split the vX from the rest of the name in ARTIC primer set description
+            description = name[:-2] + " " + name[-2:] + " primer set"
+        else:
+            description = name + " primer set"
         data.append(dict(value=name, path=bed_output_filename, description=description))
     return data
 
 
 def install_primer_file(
-    output_directory, input_filename, primer_name, primer_description
+    output_directory, input_filename, scheme_name, primer_description
 ):
-    name = re.sub(r"\W", "", str(primer_name).replace(" ", "_"))
+    name = re.sub(r"\W-", "", str(scheme_name).replace(" ", "_"))
```

Comment:
I assume you meant:
```suggestion
    name = re.sub(r"[\W-]", "", str(scheme_name).replace(" ", "_"))
```
?

Reply:
Actually neither seem to work (because this is a negative test, I need to exclude - from \W. `(?!-)\W"` works and I have switched to that.